packages/components/src/tooltip/stories/index.js

```diff
@@ -17,79 +6,80 @@ import Tooltip from '../';
 export default {
 	title: 'Components/ToolTip',
 	component: Tooltip,
+	argTypes: {
+		delay: { control: 'number' },
```

Comment:
Should we also add a `defaultValue` of `700` to keep the default value that was there when we used knobs?

```suggestion
		delay: { control: 'number', defaultValue: 700 },
```

Feel free to place it in `Default.args` if that makes more sense for you.

Reply:
In general we've been taking the direction of not re-specifying default values in Storybook, so we can showcase the actual defaults that are coming from the component code.

The props table will show descriptions and default values once this is converted to TypeScript, which will also help.